src/services/videoconference/index.js

```diff
@@ -43,319 +43,314 @@ const { ObjectId } = require('../../helper/compare');
+ * @param {[String]} usersPermissions
+ * @returns {Boolean}
+ */
+function userIsAllowedTo(permission, usersPermissions) {
+	return usersPermissions.includes(permission);
+}
 
-	static getUserRole(userPermissionsInScope) {
-		if (VideoconferenceBaseService.userIsAllowedTo(PERMISSIONS.START_MEETING, userPermissionsInScope)) {
-			return ROLES.MODERATOR;
-		}
-		if (VideoconferenceBaseService.userIsAllowedTo(PERMISSIONS.JOIN_MEETING, userPermissionsInScope)) {
-			return ROLES.ATTENDEE;
-		}
-		throw new Error('no permission to start or join a videoconference');
+function getUserRole(userPermissionsInScope) {
+	if (userIsAllowedTo(PERMISSIONS.START_MEETING, userPermissionsInScope)) {
+		return ROLES.MODERATOR;
 	}
+	if (userIsAllowedTo(PERMISSIONS.JOIN_MEETING, userPermissionsInScope)) {
+		return ROLES.ATTENDEE;
+	}
+	throw new Error('no permission to start or join a videoconference');
+}
 
-	/**
-	 *
-	 * @param {Object} app Express app
-	 * @param{*} user
-	 * @param {*} copeName
-	 * @param {* scopeId
-	 */
-	async getScopeInfo(app, user, scopeName, scopeId) {
-		let scopeTitle;
-		let event;
-		let permissionScopeId;
-		let permissionScopeName;
-		// retrieve scope information, set roomName AND scopePermissionService OR throw
-		switch (scopeName) {
-			case (SCOPE_NAMES.COURSE):
-				// fetch course metadata
-				// eslint-disable-next-line prefer-destructuring
-				permissionScopeId = scopeId;
-				permissionScopeName = 'courses';
-				scopeTitle = (await app.service(permissionScopeName).get(scopeId)).name;
-				break;
-			case (SCOPE_NAMES.EVENT):
-				// eslint-disable-next-line no-case-declarations
-				const events = (await app.service('calendar').find({
-					query: { 'event-id': scopeId },
-					payload: { userId: user.id },
-				}));
-				if (Array.isArray(events) && events.length >= 1) {
-					event = events[0];
-				} else {
-					throw new NotFound('event not found');
-				}
-
-				permissionScopeId = event['x-sc-teamId'];
-				if (!permissionScopeId) {
-					throw new NotFound('could not find videoconference enabled for this event in team');
-				}
-				permissionScopeName = 'teams';
-				scopeTitle = event.title;
-				break;
-			default:
-				throw new BadRequest('invalid scope information given');
-		}
-
-		// check permissions and set role
-		const scopePermissionService = app.service(`/${permissionScopeName}/:scopeId/userPermissions`);
-		const { [user.id]: userPermissionsInScope } = await scopePermissionService.find({
-			route: { scopeId: permissionScopeId },
-			query: { userId: user.id },
-		});
-
-		let logoutURL = `${CLIENT_HOST}/${permissionScopeName}/${permissionScopeId}`;
-		if (permissionScopeName === 'teams') logoutURL += '?activeTab=events';
-		if (permissionScopeName === 'courses') logoutURL += '?activeTab=tools';
+/**
+ *
+ * @param {Object} app Express app
+ * @param{*} user
+ * @param {*} copeName
+ * @param {* scopeId
+ */
+async function getScopeInfo(app, user, scopeName, scopeId) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['getScopeInfo' is defined but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5401970)